packages/cli-repl/src/cli-repl.ts

```diff
@@ -172,15 +171,14 @@ class CliRepl {
    * @param {NodeOptions} driverOptions - The driver options.
    */
   async requirePassword(driverUri: string, driverOptions: NodeOptions): Promise<void> {
-    const readOptions = {
-      prompt: 'Enter password: ',
-      silent: true,
-      replace: '*',
+    const passwordPromise = askpassword({
       input: this.input,
-      output: this.output
-    };
+      output: this.output,
+      replacementCharacter: '*'
+    });
+    this.output.write('Enter password: ');
     try {
-      (driverOptions.auth as any).password = await promisify(read)(readOptions);
+      (driverOptions.auth as any).password = (await passwordPromise).toString();
```

Comment:
We could keep `read` if we want to, but it doesn’t feel like it makes sense to have two different libraries for achieving the same task. We can’t use `read` for `passwordPrompt()`, though, because that doesn’t work with the REPL (because it doesn’t remove the REPL’s `.on('data')` listeners for the input).